scripts/get_oppia_classes.py

```diff
@@ -0,0 +1,196 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""get_oppia_classes.py
+This script is used to get a dictionary with directive files as keys and
+oppia classes(belonging to them uniquely) as values.
+This can be used to shift all the classes which are used only in single
+files from oppia.css to respective files
+The scripts ask for the path to directory in which you want to run it.
```

Comment:
I suggest just giving directly an example of the terminal invocation needed (see the documentation in other scripts for examples).